_applications/equoid.adoc

```diff
@@ -0,0 +1,219 @@
+
+pass:[<img src="/assets/equoid/Equoid-Software-Components.png" alt="Alt text" class="img-responsive arch" width="800px">]
+ 
+[[installation]]
+== Installation
+
+Equoid consists of a number of microservices which require deployment to an OpenShift instance. In order to get you started as swiftly as possible, in this section a walkthrough of installing and configuring the Equoid services is given. 
+
+[[prerequisites]]
+=== Prerequisites
+
+* An available OpenShift instance.
+* A terminal with the OpenShift (`oc`) command and active login session are available.
+* An OpenShift project you own.
+
+[[procedure]]
+=== Procedure
+
+. Install Image Streams and Templates
++
+Depending on your OpenShift instance, some of the image streams necessary for Equoid may not be available. To be certain you have what's necessary to build Equoid, run the following sequence of commands:
++
+....
+oc create -f https://raw.githubusercontent.com/radanalyticsio/equoid-openshift/master/fabric8-image-streams.json
+oc create -f https://radanalytics.io/resources.yaml
+oc create -f https://raw.githubusercontent.com/infinispan/infinispan-openshift-templates/master/templates/infinispan-ephemeral.json
+....
++
+These provide the Fabric8 fuse-java image stream for the data-publisher service, the radanalytics.io Oshinko resources for facilitating Spark cluster creation and deployment as well as S2I for the data-handler, and the Infinispan template for the caching microservice, respectively. 
+
+. Launch Artemis
++
+We'll get Artemis up and running, as it is necessary for both the publisher and handler:
++
+....
+oc create -f https://raw.githubusercontent.com/radanalyticsio/equoid-openshift/master/artemis-rc.yaml
+....
+
+. Launch Infinispan Cache
++
+We'll want our caching layer in place before we begin streaming and ingesting. The following will get an Infinispan pod running. `APPLICATION_USER`, `APPLICATION_PASSWORD`, `MANAGEMENT_USER` and `MANAGEMENT_PASSWORD` should be changed to your desired values. 
++
+....
+oc new-app --template=infinispan-ephemeral \
+    -l app=datagrid \
+    -p APPLICATION_NAME=datagrid \
+    -p NAMESPACE=`oc project -q` \
+    -p APPLICATION_USER=<YOUR_APP_USERNAME> \
+    -p APPLICATION_PASSWORD=<YOUR_APP_PASSWORD> \
+    -p MANAGEMENT_USER=<YOUR_MGMT_USERNAME> \
+    -p MANAGEMENT_PASSWORD=<YOUR_MGMT_PASSWORD>
+....
++
+
+. Launch Data Publisher
++
+We can next begin publishing our streaming data by starting an instance of the data publisher. `OP_MODE` described later should be set to the mode you wish. Also, `DATA_URL_PRIMARY` should be set to the location of your data set or data potential values file. 
++
```

Comment:
i think for a new user the `DATA_URL_PRIMARY` is going to throw them. imo, we could just include [this data file](https://raw.githubusercontent.com/EldritchJS/equoid-data-publisher/master/data/StockCodesLinear.txt) in our assets and then provide a default link in the box below.

i also think we should pick a default mode to start with, just to make these commands easier to cut and paste.

Reply:
Yeah, makes sense. I went with linear and the appropriate sample dataset URL that's included later in the doc where the example cases are listed. Since the options are explained I agree it's better to have an actual example instead of the {A|B|C} notation.